documentation/1.10/content/quickstart/setup.md

```diff
@@ -5,6 +5,13 @@ draft: false
 weight: 2
 ---
 
-- Download and unzip the WIT release ZIP file to a desired location.
-- OPTIONALLY: You may build the project (`mvn clean package`) to create the ZIP installer in `./imagetool/target`.
+- Download the WIT release ZIP file to a desired location.
+  - The latest ZIP file can be found on the project [releases](https://github.com/oracle/weblogic-image-tool/releases) page.
```

Comment:
The latest ZIP file --> The latest release Zip file and Zip files for earlier releases